home-connect/build.gradle

```diff
@@ -1,7 +1,7 @@
 apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-kapt'
-apply plugin: 'com.github.dcendents.android-maven'
+//apply plugin: 'com.github.dcendents.android-maven'
```

Comment:
this should not be on develop - it is only for the submodule based integration that was setup for quicker iteration